Makefile

```diff
@@ -0,0 +1,26 @@
+.PHONY: build
```

Comment:
Do we need a makefile?

`make check` is equivalent to `cargo check`, etc. `all` maybe, but is it worth maintaining this?

Reply:
It helps to run all cargo commands at once.